### DISCUSSION TITLE
Update WinGet Package

### DISCUSSION THREAD

**Discussion by Hexality** [transmission/transmission#5022] [Ideas]
Transmission.Transmission package on the WinGet Repository has baeen abandoned for almost 3 years, will we get 4.0.1 and future updates there?
I wanted to do the update there myself but I don't think I can simply upload there manifest for programs I don't either own or work for.

[manifests.zip](https://github.com/transmission/transmission/files/10819376/manifests.zip)

[microsoft/winget-pkgs](https://github.com/microsoft/winget-pkgs/)

**Comment by nevack** [comment]
I think you can try to recreate previous update 
https://github.com/microsoft/winget-pkgs/commit/5a5454530b60bfe3247d25dc392dd88e853d9d27

**Reply by Hexality** [reply]
I did try, but it requested me to sign the terms there and I don't think I can sign the PR terms without owning or being part of the project.
Included the manifests.zip because of that.
